src/util.rs

```diff
@@ -0,0 +1,11 @@
+extern crate regex;
```

Comment:
Don't need to declare crate when using 2018 edition.

Reply:
Done in 5e03a24 :-)